synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -172,6 +172,20 @@ Widget_Timeslider::Widget_Timeslider():
 			  | Gdk::BUTTON_RELEASE_MASK
 			  | Gdk::BUTTON_MOTION_MASK
 			  | Gdk::SCROLL_MASK );
+
+
+	GdkPixbuf *lower_boundary_pixbuf_c = gtk_icon_theme_load_icon (gtk_icon_theme_get_default (),
```

Comment:
Why do not use Gtkmm methods here?
```
auto icon_theme = Gtk::IconTheme::get_default();
lower_boundary_pixbuf = icon_theme->load_icon("bound_button_icon_lower", 16);
upper_boundary_pixbuf = icon_theme->load_icon("bound_button_icon_upper", 16);
```

Reply:
oh yes definitely, for some reason I was having some trouble getting the equivalent gtkmm methods right away, so I thought to do it with gtk first then switch over but seems like I forgot xD.

Reply:
hmm, the methods work but surprisingly I couldn't find them in either the gtkmm 3.0 docs or the 4.0 ones.

Reply:
https://developer-old.gnome.org/gtkmm/3.24/classGtk_1_1IconTheme.html#a9e9307cefa458eea1e8946d9def2e527